src/sdp.ts

```diff
@@ -99,15 +111,19 @@ a=max-message-size:100000
 a=candidate:1467250027 1 UDP 1467250027 ${host} ${port} typ host\r\n`
 }
 
-// Create an answer SDP from a multiaddr
+/**
+ * Create an answer SDP from a multiaddr
+ */
 export function fromMultiAddr (ma: Multiaddr, ufrag: string): RTCSessionDescriptionInit {
   return {
     type: 'answer',
     sdp: ma2sdp(ma, ufrag)
   }
 }
 
-// Replace the ufrag and password values in a SDP
+/**
+ * Replace (munge) the ufrag and password values in a SDP
```

Comment:
Might just be me reading functions that I require pre-knowledge for, but I stil feels there is a lot of guess work required on my end to get what this function does without reading the actual implementation.